flask_app.py

```diff
@@ -34,41 +35,45 @@
 
 db = SQLAlchemy(app)
 
-#class Comment(db.Model):
+# class Comment(db.Model):
 #
 #    __tablename__ = "comments"
 #
 #    id = db.Column(db.Integer, primary_key=True)
 #    content = db.Column(db.String(4096))
 
-#@app.route('/')
-#def hello_world():
+# @app.route('/')
+# def hello_world():
 #    return 'Hello from Flask!'
 
-#@app.route('/')
-#def index():
+# @app.route('/')
+# def index():
 #    return render_template("main_page.html")
 
-#@app.route("/", methods=["GET", "POST"])
+
+# @app.route("/", methods=["GET", "POST"])
 @app.route("/")
 def index():
-    from forecastSrc.forecast_db_interface import BbcTable, OwmTable, YrTable, VollTable, forecast_db_interface
+    from forecastSrc.forecast_db_interface import (BbcTable, OwmTable,
+                                                   VollTable, YrTable,
+                                                   forecast_db_interface)
+
     if request.method == "GET":
         dbInterface = forecast_db_interface()
-        renderedRetunVal = render_template  (
-                                    "main_page.html"
-                                    ,bbcData=BbcTable.query.all()
-                                    ,owmData=OwmTable.query.all()
-                                    ,yrData=YrTable.query.all()
-                                    ,vollData=VollTable.query.all()
-                                    ,day1Table=dbInterface.dayAwayTable()
-                                    ,day2Table=dbInterface.dayAwayTable(numDays=2)
-                                )
+        renderedRetunVal = render_template(
+            "main_page.html",
```

Comment:
I do prefer this style of code-writing over the other, where the comma is at the end of the line. Most people add a new argument to a function either in the middle of other arguments, or at the end of all the arguments. Very seldom do people add a new argument at the top of the argument list. So in this case, if a new argument is added at the end, 2 lines would have to be touched instead of just 1.

This is just a convention thing and while I agree that both conventions are ok, my own preference is towards the convention where the commas are at the start of the line. Could you please revert this change for the pull request?